spec/errors.md

```diff
@@ -335,3 +307,65 @@ or an internally inconsistent set of options.
 > Your {$field} is {$id :get field=$field}
 > ```
 
+### Bad Operand
```

Comment:
Should there also be a `Missing Operand` (or `Missing Option`) error for cases where a required operand (or option) is missing (`{:date}`) or null?

Reply:
I'm not so sure; it seems like a missing operand is a Bad Operand.